pagopa/pn-frontend
# manage upload of big size files - pn-7268

## Short description
In New notification page, step Attachments, front-end freezes if we try to upload big files. 

## List of changes proposed in this pull request
- Mooved the calculation of Unit8Array to the api call

## How to test
Go to new notification page, upload a big file and check that the notification creation goes well (try the file in the linked issue)
